extensions/cache/deployment/src/main/java/io/quarkus/cache/deployment/devconsole/CacheDevUiConsoleProcessor.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.cache.deployment.devconsole;
+
+import io.quarkus.cache.runtime.devconsole.CacheJsonRPCService;
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.pkg.builditem.CurateOutcomeBuildItem;
+import io.quarkus.devui.spi.JsonRPCProvidersBuildItem;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.Page;
+
+public class CacheDevUiConsoleProcessor {
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    CardPageBuildItem create(CurateOutcomeBuildItem bi) {
+        CardPageBuildItem pageBuildItem = new CardPageBuildItem("Cache");
+        pageBuildItem.addPage(Page.webComponentPageBuilder()
+                .title("Caches")
+                .componentLink("cache-component.js")
+                .icon("font-awesome-solid:database"));
+
+        return pageBuildItem;
+    }
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    JsonRPCProvidersBuildItem createJsonRPCServiceForCache() {
+        return new JsonRPCProvidersBuildItem("Caches", CacheJsonRPCService.class);
```

Comment:
The name used here should be Cache (same as what is used in the CardPageBuildItem and the name in the extension yaml.